### PR TITLE
mneESTIMATEtreeitem -> mneDATAtreeitem

### LINKED ISSUE #370: Rename MNEEstimateTreeItem in Disp3D
Rename MNEEstimateTreeItem to MNEDataTreeItem. This includes renaming the header and cpp files as well. https://github.com/mne-tools/mne-cpp/blob/f7a125f467fe745baa54615b9044bc35d06b3bbf/libraries/disp3D/engine/model/items/sourcedata/mneestimatetreeitem.h#L109

### PR DESCRIPTION
**Pull request by juangpc**
Hi. 

Not used to qt/cpp here.  

I've a branch here to try to rename the class throughout the code. By now: 
 - Renamed the class file and its header.
 - Changed all its references throughout the code. 
 - Built for release, debug and profiling configs and all 3 were fine. (lots of warnings but OK).
Although I found the 
```
"Error: dependent '..\ some-file' does not exist." 
```
To solve it I had to "Run qmake" before clean and build.

I would feel much better if someone could try the code itself. I would propose either: 
 - someone to fork it from me, execute and try a few examples, fix and PR if something is wrong.
 - (probably better) to (i) create a branch in mne-tools/mne-cpp from master and accept the PR there. And (ii) someone to fetch that branch and do the testing there, etc... until eventually merge it into master.

    Juan

Closes #370

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,6 @@ Makefile
 .project
 .settings
 .vscode
+
```
Comment (LorenzE) [inline .gitignore]: Pls remove .gitignore from versioned files.

File: applications/mne_analyze/extensions/music/music.cpp [hunk 2]
```diff
@@ -264,7 +264,7 @@ void Music::calculate()
 //    p3DDataModel->addSurfaceSet(parser.value(subjectOption), evoked.comment, t_surfSet, t_annotationSet);
 
 //    //Add rt source loc data and init some visualization values
-//    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption), evoked.comment, sourceEstimate, t_clusteredFwd)) {
+//    if(mnedatatreeitem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption), evoked.comment, sourceEstimate, t_clusteredFwd)) {
```
Comment (LorenzE) [inline applications/mne_analyze/extensions/music/music.cpp]: `MNEDataTreeItem`

File: applications/mne_scan/libs/scDisp/realtimesourceestimatewidget.h [hunk 3]
```diff
@@ -152,7 +152,7 @@ class SCDISPSHARED_EXPORT RealTimeSourceEstimateWidget : public MeasurementWidge
     QSharedPointer<SCMEASLIB::RealTimeSourceEstimate>   m_pRTSE;                /**< The real-time source estimate measurement. */
     QSharedPointer<DISP3DLIB::SourceEstimateView>       m_pSourceEstimateView;  /**< The 3D source estimate view. */
 
-    QPointer<DISP3DLIB::MneEstimateTreeItem>            m_pRtItem;              /**< The Disp3D real time items. */
+    QPointer<DISP3DLIB::MneDataTreeItem>            m_pRtItem;              /**< The Disp3D real time items. */
```
Comment (LorenzE) [inline applications/mne_scan/libs/scDisp/realtimesourceestimatewidget.h]: Mind indentation

File: examples/ex_clustered_inverse_mne/main.cpp [hunk 4]
```diff
@@ -293,7 +293,7 @@ int main(int argc, char *argv[])
 
     p3DDataModel->addSurfaceSet(parser.value(subjectOption), "MRI", t_surfSet, t_annotationSet);
 
-    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
+    if(MneDataTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
```
Comment (LorenzE) [inline examples/ex_clustered_inverse_mne/main.cpp]: Mind indentation

File: examples/ex_clustered_inverse_mne_raw/main.cpp [hunk 5]
```diff
@@ -650,7 +650,7 @@ int main(int argc, char *argv[])
 
     p3DDataModel->addSurfaceSet(parser.value(subjectOption), "MRI", t_surfSet, t_annotationSet);
 
-    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
+    if(MneDataTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
```
Comment (LorenzE) [inline examples/ex_clustered_inverse_mne_raw/main.cpp]: Mind indentation

File: examples/ex_clustered_inverse_pwl_rap_music_raw/main.cpp [hunk 6]
```diff
@@ -579,7 +579,7 @@ int main(int argc, char *argv[])
     p3DDataModel->addSurfaceSet(parser.value(subjectOption), evoked.comment, t_surfSet, t_annotationSet);
 
     //Add rt source loc data and init some visualization values
-    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
+    if(MneDataTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
```
Comment (LorenzE) [inline examples/ex_clustered_inverse_pwl_rap_music_raw/main.cpp]: Mind indentation

File: examples/ex_clustered_inverse_rap_music_raw/main.cpp [hunk 7]
```diff
@@ -535,7 +535,7 @@ int main(int argc, char *argv[])
     p3DDataModel->addSurfaceSet(parser.value(subjectOption), evoked.comment, t_surfSet, t_annotationSet);
 
     //Add rt source loc data and init some visualization values
-    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
+    if(MneDataTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
```
Comment (LorenzE) [inline examples/ex_clustered_inverse_rap_music_raw/main.cpp]: Mind indentation pls check the rest of your changes as well

File: libraries/disp3D/disp3D.pro [hunk 8]
```diff
@@ -91,7 +91,6 @@ SOURCES += \
     engine/model/items/freesurfer/fsannotationtreeitem.cpp \
     engine/model/items/hemisphere/hemispheretreeitem.cpp \
     engine/model/items/sourcespace/sourcespacetreeitem.cpp \
-    engine/model/items/sourcedata/mneestimatetreeitem.cpp \
```
Comment (LorenzE) [inline libraries/disp3D/disp3D.pro]: Pls keep order of the files in tact. Just rename the .cpp and .h file names in place here.

File: .gitignore (hunk 1 above)
Comment (juangpc) [reply .gitignore]: done

File: examples/ex_clustered_inverse_mne/main.cpp (hunk 4 above)
Comment (juangpc) [reply examples/ex_clustered_inverse_mne/main.cpp]: Sorry, don't see what is wrong here.

Comment (LorenzE) [reply examples/ex_clustered_inverse_mne/main.cpp]: The passed functions parameter are seperated by linebreaks and therefore should be indented to start at the function's bracket `(`

**Comment by juangpc** [comment]
Hi, 

- Indentations solved. Thanks for your comment.
- Order of files in libraries/disp3D/disp3D.pro solved (I think). Although this change was actually done by qmake itself. So, I'm not sure if it will persist.
- music.cpp also fixed. Nice catch. :)

**Comment by juangpc** [comment]
Hi, 

What else should I address? Thanks.

**Pull request merged**